app/lib/cooldown.js

```diff
@@ -6,8 +6,15 @@ module.exports = Base => class extends Base {
         
         //object containing the cooldown times for the commands. new commands would need a new property added here.
         this.coodownTimes = {
-            'home': 1000 * 60 * 15,
-            'warp': 1000 * 60 * 15
+            '!home': 1000 * 60 * 15,
+            '!warp': 1000 * 60 * 15,
+            'Location teleporting': 1000 * 60 * 30
```

Comment:
We didn't really discuss this with your last PR but why 30 minutes not 15 like the rest?